docs/src/index.md

```diff
@@ -26,3 +26,33 @@ M = Sphere(2)
 γ = shortest_geodesic(M, [0., 0., 1.], [0., 1., 0.])
 γ(0.5)
 ```
+
+## Citation
+
+If you use `Manifolds.jl` in your work, please cite the following
+
+```biblatex
+@online{2106.08777,
```

Comment:
Are you sure `@online` and `@software` are BibTeX styles? I think for accessibility we should avoid extensions some document types may not support.

Reply:
They are BibLaTeX styles actually. The point is, for bibtex we could only use `@misc` then in both cases.

Reply:
I'm just a little worried because the couple of times I tried to use BibLeTeX it didn't work with the document style I had to use.

Reply:
I see your point.
The problem is, that _none_ of the bibtex styles actually features some preprint ideas, so the fields `eprinttype` and `print` are also exclusively bib latex; in bibtex you have to “cheat”, see for example https://tex.stackexchange.com/a/251051/8042.
So I would prefer not to use bibtex when possible, but the much more modern BibLaTeX styles. We could mention that these are BibLaTeX, if you want?

Reply:
OK, let's just point out that these are BibLaTeX styles.

Reply:
I added a note and collected them also in a CITATION.bib, which I saw in several other packages.